content/en/contribute/docs/style-guide.md

```diff
@@ -233,16 +233,16 @@ This is a sample note.
 
 ### Images
 
-Use the figure shortcode for images. This uses built-in styling that is responsive.
+The figure shortcode is preferred for images as it uses built-in styling that is responsive, however Markdown syntax is also acceptable. Do not use html styling.
 
 | Do | Don't |
 |---|---|
-| `{{figure src="image.png" class="right col-6 col-lg-3"}}` | `<img src="image.png" width="30%" align="right">` |
+| `{{< figure src="image.png" class="right col-6 col-lg-3" >}}` | `<img src="image.png" width="30%" align="right">` |
 
 The image can also link to a file by using the link attribute.
```

Comment:
```suggestion
It is good practice for the image to link to the image file so that a larger version can be viewed easily. This can be done using the `link` attribute with the `figure` shortcode, which is less error prone than adding a link to the markdown image notation.
```